### ISSUE TITLE
Crash on Python 2.7.13

### ISSUE THREAD

**Issue by remram44** [ViDA-NYU/reprozip#247]
Looks like yet another weird internal breaking change from Python:
```
Traceback (most recent call last):
  File "/usr/local/bin/reprounzip", line 11, in <module>
    load_entry_point('reprounzip', 'console_scripts', 'reprounzip')()
  File "/usr/src/app/reprozip/reprounzip/reprounzip/main.py", line 141, in main
    args.func(args)
  File "/usr/src/app/reprozip/reprounzip-docker/reprounzip/unpackers/docker.py", line 119, in docker_setup
    docker_setup_build(args)
  File "/usr/src/app/reprozip/reprounzip/reprounzip/unpackers/common/misc.py", line 65, in wrapper
    return func(args)
  File "/usr/src/app/reprozip/reprounzip-docker/reprounzip/unpackers/docker.py", line 408, in docker_setup_build
    for k, v in iteritems(os.environ)
TypeError: descriptor 'iteritems' requires a 'dict' object but received a 'instance'
```

**Comment by remram44** [comment]
The only affected piece of code (calling `dict.iteritems(os.environ)`) seems to be here: 256456de.

**Comment by remram44** [comment]
Fixed with a11e68c6.
